### PR TITLE
Badge

### PR DESCRIPTION
**Pull request by anandsr21harshit**

### CODE REVIEW

File: Components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    width: 2em;
+}
+
+.message-icon{
+    width: 2em;
+}
+
+.notification{
+    position: absolute;
+    background-color: red;
```
Comment (Logan1x) [inline Components/Badge/badge.css]: Can you please declare gloabl variables for this color and use that at this place?

**Review by Logan1x — APPROVED** [review]
change that color property and you are good to go.

File: Components/Badge/badge.css (hunk 1 above)
Comment (anandsr21harshit) [reply Components/Badge/badge.css]: Done

File: Components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.notification{
+    position: absolute;
+    background-color: var(--notif-background);
+    color: var(--text-color);
+    border-radius: 50%;
+    font-size: 0.8rem;
+    width: 25%;
+    height: 25%;
+    min-width: 15px;
```
Comment (Shah-Arsalan) [inline Components/Badge/badge.css]: You can use rems for consistent use of units

**Review by Shah-Arsalan — APPROVED** [review]

**Comment by Logan1x** [comment]
You are good to go @anandsr21harshit

File: Components/Badge/badge.css (hunk 2 above)
Comment (anandsr21harshit) [reply Components/Badge/badge.css]: Changes done

**Pull request merged**